README.md

```diff
@@ -1,4 +1,4 @@
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/0642a6f7d92542e39f2dd68139cd5586)](https://www.codacy.com/gh/Seagate/cortx-hare/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-hare&amp;utm_campaign=Badge_Grade)
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/0642a6f7d92542e39f2dd68139cd5586)](https://www.codacy.com/gh/Seagate/cortx-hare/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-hare&amp;utm_campaign=Badge_Grade) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/pujamudaliar/cortx-hare-1/blob/add-license-slack-badge/LICENSE) [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
```

Comment:
Sorry, but the link with Apache 2.0 badge points to a private repository: `https://github.com/pujamudaliar/cortx-hare-1/blob/add-license-slack-badge/LICENSE`

Is it intended?

Reply:
Perhaps it is possible to use relative paths in the links?

Reply:
@knekrasov : Thanks for pointing that out.Since I was on my branch,I could have opened the link from my branch.Changes are not been committed. Please check if it is fine.